functions.php

```diff
@@ -1353,3 +1353,56 @@ function wp_autoupdates_toggle_auto_updates() {
 	wp_send_json_success();
 }
 add_action( 'wp_ajax_toggle_auto_updates', 'wp_autoupdates_toggle_auto_updates' );
+
+/**
+ * Set up auto-updates help tabs.
+ */
+function wp_autoupdates_help_tab_update_core() {
+	$screen = get_current_screen();
+
+	if ( 'update-core' === $screen->id || 'update-core-network' === $screen->id ) {
+		$help_tab_content  = '<p>' . __( 'Plugins and Themes with auto-updates enabled will display the estimated date of the next auto-update. Auto-updates schedule depends on WordPress planned tasks.' ) . '</p>';
+		$help_tab_content .= '<p>' . __( 'Please note: WordPress auto-updates may be overridden by third-parties plugins or custom code.' ) . '</p>';
+		$help_tab_content .= '<p>' . sprintf(
```

Comment:
In core, it would probably be more appropriate to have the link in the help sidebar (above the "Support" link), rather than in the tab content.

It's possible to _almost_ do that in the plugin by hooking into `in_admin_header` instead of `current_screen`, by which time core will have set the sidebar content and we could append to it.  Doing that would make the link come **after** the "Support" link, but I think that's good enough at this point.

Reply:
Yep, thanks for the heads up. I'll add an inline comment :-)

Reply:
Added few `@todo` inline comments